src/tools/PrizeTierController/SavePrizeTiersModal/PrizePoolTransactionDisplay.tsx

```diff
@@ -58,16 +59,19 @@ export const PrizePoolTransactionDisplay = (props: {
   if (edits.edited) {
     return (
       <li className='bg-pt-purple-dark p-3 rounded-xl'>
-        <PrizePoolTitle prizePool={prizePool} className='mb-4 pb-2 border-b' />
+        <PrizePoolTitle
+          prizeTierHistoryContract={prizeTierHistoryContract}
+          className='mb-4 pb-2 border-b'
+        />
         <div className='flex gap-2 opacity-60 text-xxs mb-2'>
           Copy Raw Config
           <CopyIcon text={rawPrizeTierString} />
         </div>
         <div>
-          {(!transaction || transaction.state === TransactionState.pending) && (
+          {(!transaction || transaction?.response?.hash === undefined) && (
```

Comment:
What happens if the transaction fails?

Reply:
If it fails before submission (most wallets check for malformed data, etc.) the button will simply re-appear after an error popup on the bottom right.

If the transaction fails on-chain, it should still return a transaction hash and the receipt will be displayed - that can then be investigated further. In practice, no failing TX should be possible to get submitted.

I'll re-add the pending state check though since those can co-exist.